### PR TITLE
port select-date to next

### LINKED ISSUE #6: Port Select Time Page

### PR DESCRIPTION
**Pull request by Fredmpk**
imported and adjusted select-date and necessary components(display-dates, labels, time-slots) to next. Also created time-selection to keep page static

closes #6

### CODE REVIEW

**Comment by Fredmpk** [comment]
closes #6

File: src/app/(full-page)/movies/[movieId]/time/page.tsx [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
+        <div className="text-white text-base text-center">
+          Select Date & Time
+        </div>
+        <div className="text-white-dimmed text-sm py-2 px-2 font-bold">
+          DATE
+        </div>
+        <TimeSelection />
+      </div>
+
+      <Link href={`../../movies/${movieId}/select-seats`}>
```
Comment (dnmct) [inline src/app/(full-page)/movies/[movieId]/time/page.tsx:39]: This href is not correct

File: src/app/(full-page)/movies/[movieId]/time/page.tsx [hunk 2]
```diff
@@ -1,11 +1,48 @@
...
   params: { movieId: string };
 }) {
+  const { movieId } = params;
+
   return (
-    <div>
-      <span>Booking Time - {params.movieId}</span>
+    <div className="flex flex-col bg-dark h-dvh px-5 py-8">
+      <div className="flex-col w-full items-center relative mb-8">
+        <Link href={`../../movies/${movieId}`}>
```
Comment (dnmct) [inline src/app/(full-page)/movies/[movieId]/time/page.tsx:14]: This href is not correct, don't use relative paths

**Review by dnmct — APPROVED** [review]
Very nice work Fred

**Pull request merged**